examples/juniper/README.md

```diff
@@ -0,0 +1,34 @@
+# How to: verify and use the services on cPTX
+
+This examples shows about how to verify and use the GRPC services on the cPTX
+
+## Get into cPTX and check the status of GRPC services
+
+Once the KNE cluster is up, get into cPTX and Check the status of GRPC services
+
+```bash
+kubectl exec -it -n <NAMESPACE> cptx1 -- cli
+root@cptx1>
+root@cptx1> show system connections |match 50051
+tcp6       0      0 :::50051                :::*                    LISTEN      XXXXX/jsd
+
+root@cptx1>
+root@cptx1> show configuration system services extension-service
+request-response {
+    grpc {
+        ssl {
+            hot-reloading;
+            use-pki;
+        }
+    }
+}
+root@cptx1>
+```
+
+## Dial the services using cli tools
+
+```bash
+export GNMI_USER=root
+export GNMI_PASS=Google123
+gnmi_cli -a <NODE EXTERNAL IP>:50051 -q "/interfaces/interface/" -tls_skip_verify -with_user_pass
```

Comment:
I think this port should be 9339 based on the outside port assignment in the example.

Can you add a line about how to use kubectl to check all of the services are exposed correctly? Maybe moving this README directly into cptx-host will be more useful as the exact namespace and ports can be used

Also can you add examples for the other services as well? Seems verification is the same since they all run on the same port

here is an example that has proven very helpful to our internal users: https://learn.srlinux.dev/tutorials/infrastructure/kne/srl-with-oc-services/

Reply:
Oops, good catch. Sure, I will send another change shortly.

Reply:
Done. Also, added more details to README.